README.md

```diff
@@ -137,6 +137,29 @@ class YourRepository extends Repository {
 
 ## Retrieving and Persisting Data
 
+### `findAll`
```

Comment:
The documentation needs to be updated to reflect the current implementation

Reply:
Updated.